### PR TITLE
Food update

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
NEW FOODS, some even have nice little healing chems in them.

## Why It's Good For The Game
Gives clerks more lc13 based food making options.

## Changelog
:cl:
add: Wormfood, credit for the sprite goes to a Ghost#7213.
add: violet and amber steaks using sprites by TotallyAMokoi#4946
fix: sprites for ordeal butchery.
tweak: pbird sugar cookie sprite for sugar cookies by TotallyAMokoi#4946
tweak: changed the ordeal meat from human/mutant to just meat.
tweak: replaced highly toxic slime jelly in raw orange ordeal meat with yuck. Its funnier to vomit violently than take silent toxin damage.
/:cl:

### CODE REVIEW

File: code/game/objects/items/food/meat.dm [hunk 1]
```diff
@@ -929,6 +897,45 @@
...
+
+/obj/item/food/meat/slab/fruit
+	name = "curious meat"
+	desc = "Wretched rubbery meat cut from a gigantic heart. It tastes of anxiety and has the texture of uncooked squid."
+	icon_state = "fruitmeat"
+	food_reagents = list(/datum/reagent/consumable/nutriment/protein = 4, /datum/reagent/consumable/sugar = 3)
+	tastes = list("tender meat" = 1, "jelly" = 1, "slime" = 1)
+	foodtypes = MEAT | RAW | TOXIC | JUNKFOOD
+
+/obj/item/food/meat/slab/human/mutant/fruit/MakeGrillable()
```
Comment (EgorDinamit) [inline code/game/objects/items/food/meat.dm:917]:
```suggestion
/obj/item/food/meat/slab/fruit/MakeGrillable()
```

Comment (EgorDinamit) [inline code/game/objects/items/food/meat.dm:917]:
```suggestion
/obj/item/food/meat/slab/fruit/MakeGrillable()
```

File: code/game/objects/items/food/pastries.dm [hunk 2]
```diff
@@ -637,7 +637,7 @@
 /obj/item/food/cookie/sugar
 	name = "sugar cookie"
 	desc = "Just like your little sister used to make."
-	icon_state = "sugarcookie"
+	icon_state = "sugarcookie_pbird"
```
Comment (EgorDinamit) [inline code/game/objects/items/food/pastries.dm:640]: Maybe just make a subtype of sugar cookies?

Comment (InsightfulParasite) [reply code/game/objects/items/food/pastries.dm:640]: Seems a bit redundant though.

File: ModularTegustation/tegufood.dm [hunk 3]
```diff
@@ -22,3 +22,48 @@
...
+	tastes = list("crunchy popping" = 1, "consumption" = 1)
+	foodtypes = MEAT | GROSS
+
+/obj/item/food/wormfood/Initialize()
+	. = ..()
+	if(prob(50))
+		new /obj/item/food/wormfood(get_turf(src))
+		qdel(src)
+
+/obj/item/food/wormfood //heals around 7 damage when consumed
```
Comment (EgorDinamit) [inline ModularTegustation/tegufood.dm:63]: This one is unused. Is it intentional?

File: ModularTegustation/tegufood.dm [hunk 4]
```diff
@@ -22,3 +22,48 @@
...
+	icon_state = "wormfood1"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/drug/maint/tar = 1, /datum/reagent/yuck = 2)
+	tastes = list("crunchy popping" = 1, "consumption" = 1)
+	foodtypes = MEAT | GROSS
+
+/obj/item/food/wormfood/Initialize()
+	. = ..()
+	if(prob(50))
+		new /obj/item/food/wormfood(get_turf(src))
+		qdel(src)
```
Comment (EgorDinamit) [inline ModularTegustation/tegufood.dm:61]: I think you meant to use another type in either definition or when creating new food.

Comment (EgorDinamit) [inline ModularTegustation/tegufood.dm:61]: I think you meant to use another type in either definition or when creating new food.

File: ModularTegustation/tegufood.dm (hunk 3 above)
Comment (InsightfulParasite) [reply ModularTegustation/tegufood.dm:63]: No this was a genuine mistake

**Review by EgorDinamit — APPROVED** [review]

**Pull request merged**